command-expansion-server/src/lib/codegen/CommandTypeCodegen.ts

```diff
@@ -20,18 +20,20 @@ function generateTypescriptCode(dictionary: ampcs.CommandDictionary): {
   // language=TypeScript
   const declarations = `
 declare global {
-${typescriptFswCommands.map((fswCommand) => fswCommand.declaration).join("\n")}
+${typescriptFswCommands.map((fswCommand) => fswCommand.declaration).join('\n')}
 \tconst Commands: {\n${dictionary.fswCommands
     .map((fswCommand) => `\t\t${fswCommand.stem}: ${fswCommand.stem},\n`)
-    .join("")}\t};
+    .join('')}\t};
 }`;
 
   // language=TypeScript
   const values = `
-${typescriptFswCommands.map((fswCommand) => fswCommand.value).join("\n")}
+${typescriptFswCommands.map((fswCommand) => fswCommand.value).join('\n')}
 export const Commands = {${dictionary.fswCommands
     .map((fswCommand) => `\t\t${fswCommand.stem}: ${fswCommand.stem},\n`)
-    .join("")}};
+    .join('')}};
+const globalThis = (0,eval)("this");
```

Comment:
@mattdailis and I discovered this is unnecessary, so we can remove this line.

for reference, here is where this was concocted from https://stackoverflow.com/questions/9107240/1-evalthis-vs-evalthis-in-javascript/9107367#9107367 but proves unnecessary